products/idn/api/patch-guide.md

```diff
@@ -0,0 +1,474 @@
+---
+id: patch-guide
+title: Patch Guide
+pagination_label: Patch Guide
+sidebar_label: Patch Guide
+sidebar_position: 8
+sidebar_class_name: patchGuide
+keywords: ['patch']
+description: Read this guide to learn how to send PATCH requests. 
+tags: ['patch']
```

Comment:
add a tag of 'guide' here as well. I think it will one day in the not-so-distant future to be able to find all of our guides in our docs easily by their tag.